x-pack/test/security_solution_api_integration/test_suites/detections_response/detection_engine/rule_gaps/trial_license_complete_tier/manual_rule_run.ts

```diff
@@ -0,0 +1,317 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from 'expect';
+
+import moment from 'moment';
+import { FtrProviderContext } from '../../../../../ftr_provider_context';
+import { getCustomQueryRuleParams, scheduleRuleRun } from '../../../utils';
+import {
+  createAlertsIndex,
+  deleteAllRules,
+  createRule,
+  deleteAllAlerts,
+} from '../../../../../../common/utils/security_solution';
+
+const buildSchedule = (
+  startDate: moment.Moment,
+  endDate: moment.Moment,
+  intervalInMinutes: number
+) => {
+  const schedule = [];
+  const interval = `${intervalInMinutes}m`;
+  let currentDate = startDate.clone();
+  while (currentDate.isBefore(endDate)) {
+    schedule.push({
+      interval,
+      run_at: currentDate.add(intervalInMinutes, 'm').toISOString(),
+      status: 'pending',
+    });
+    currentDate = currentDate.clone();
+  }
+  return schedule;
+};
+
+export default ({ getService }: FtrProviderContext) => {
+  const supertest = getService('supertest');
+  const log = getService('log');
+  const es = getService('es');
+
+  describe('@ess @serverless @skipInServerlessMKI manual_rule_run', () => {
```

Comment:
can we add comment why tests skipped in MKI?

Reply:
done